docs/usage/self-hosted-experimental.md

```diff
@@ -1,22 +1,23 @@
-# Self-Hosted Experimental Options
+# Self-hosted experimental options
 
-The following options are considered "experimental" in that:
+The following options are "experimental" because:
 
-- They should not commonly be needed
-- They are typically an effort to work around some other service or platform's problem
-- We hope they can each be removed one day, or we are waiting to be sure they work as intended
+- They are not commonly needed
+- They are typically an effort to work around some other service's or platform's problem
+- They can be removed at any time
+- They are options for Renovate's internal use to validate they work as intended
 
-Experimental options which are commonly used and for which there is no external solution in sight should be replaced with official options.
+Experimental options which are commonly used and for which there is no external solution in sight should be replaced with official options ("Who is doing the replacing here? Renovate bot team or end-user?").
 
-Please use experimental options at your own risk - they may be removed or behavior changed in any release, not just major, however all attempts will be made not to inconvenience anyone.
+Use experimental options at your own risk - they may be removed or behavior changed in any release, not just major, however all attempts will be made not to inconvenience anyone. (Aren't we breaking our "We always follow SemVer promise here?")
 
 ## GITLAB_IGNORE_REPO_URL
 
-If set to any value, Renovate will ignore the Project's `http_url_to_repo` value and instead construct the git URL manually.
+If set to any value, Renovate will ignore the Project's `http_url_to_repo` value and instead construct the Git URL manually.
 
 ## RENOVATE_CACHE_NPM_MINUTES
 
-This must be a number, and would override Renovate's default npm cache time of 15 minutes for the npm datasource.
+If set to any integer, Renovate will use this integer instead of the default npm cache time (15 minutes) for the npm datasource.
```

Comment:
Or do we actually accept any valid type `Number` here? Would `1.5` or `1,55` work properly?

Reply:
https://github.com/renovatebot/renovate/blob/7d6870a6d06f0797520fb339120c7aaa4a1d3407/lib/datasource/npm/get.ts#L216-L218

must be int

Reply:
Cheers, thanks for checking that in the code. 😄